commit 27e159a06042fd144dd9d30183cb38b1eba943d7
Author: Aleksei Nikiforov <darktemplar@basealt.ru>
Date:   Fri Jun 25 11:36:29 2021 +0300

    Fix crash in vtkOpenSlideReader destructor due to uninitialized pointer use

diff --git a/Domains/Microscopy/vtkOpenSlideReader.h b/Domains/Microscopy/vtkOpenSlideReader.h
index 0e6daf8667..18a767886f 100644
--- a/Domains/Microscopy/vtkOpenSlideReader.h
+++ b/Domains/Microscopy/vtkOpenSlideReader.h
@@ -73,7 +73,7 @@ protected:
   void ExecuteDataWithInformation(vtkDataObject* out, vtkInformation* outInfo) override;
 
 private:
-  openslide_t* openslide_handle;
+  openslide_t* openslide_handle = nullptr;
 
   vtkOpenSlideReader(const vtkOpenSlideReader&) = delete;
   void operator=(const vtkOpenSlideReader&) = delete;
